server/src/main/java/com/objectcomputing/checkins/services/skills/Skill.java

```diff
@@ -43,6 +42,11 @@
     @Schema(description = "the skill is extraneous (or not)", required = true)
     private boolean extraneous = false;
 
+    @ManyToMany(mappedBy = "skills",
+            fetch = FetchType.EAGER,
+            cascade = CascadeType.ALL)
+    private List<SkillTag> tags;
```

Comment:
Should tags be exclusive to skills? Perhaps they should be able to tag other things? Also does the `@ManyToMany` have an effect?

Reply:
Rereading the story it says "create service for tagging skills" and I took that to mean the tags were for skills but that's definitely not what it says. This would be a very do-able refactor but it would be more involved than I'd want to do since I'm not on this project anymore. My vote is we should leave it as skill tags for now and make a story to refactor it to be more generalized.

As for the many-to-many JDBC can use it to auto-populate based on the join using the @join annotation in the repository and the matching annotations on the SkillTag entity.